README.md

```diff
@@ -152,6 +135,11 @@ lang-exclusive: ['en', 'fr']
 ---
 ```
 
+#### Machine-aware site building
+_New in 1.5.0_
+
+Polyglot will only start builds after it confirms there is a cpu core ready to accept the build thread. This ensures that jekll will build large sites efficiently streamlining build processes instead of overloading machines with process thrash.
```

Comment:
@untra this description is slightly misleading. it doesn't actually check whether the cpu core is *available* (i.e. not busy), as it could be used by another process. it just makes sure it doesn't run more child processes at a time than there are cpu cores.

Reply:
you'd still have issues if you e.g. build multiple polyglot sites at the same time (w/o assigning a subset of cpus to the parent process, which would a the solution for that).